ducttape/utils.py

```diff
@@ -201,6 +201,10 @@ def _get_chrome_driver(self, download_location, headless, chrome_option_prefs):
         if chrome_option_prefs:
             prefs.update(chrome_option_prefs)
         chrome_options.add_experimental_option('prefs', prefs)
+        
+        # when run from a Docker container
```

Comment:
Hmm doing a little reading makes me a bit nervous about always having the `--no-sandbox` command on. 

Is your thought that the risk is mitigated given that in this package we're always going to specific, trusted (hopefully) websites?